unstructured/ingest/runner/confluence.py

```diff
@@ -13,8 +13,8 @@ def run(
         url: str,
         user_email: str,
         api_token: str,
-        max_num_of_spaces: int,
-        max_num_of_docs_from_each_space: int,
+        max_num_of_spaces: int = 500,
+        max_num_of_docs_from_each_space: int = 100,
```

Comment:
same as above

Reply:
should we have cli.cmds.confluence import these defaults from the runner so they are only explicitly assigned in one place? even if it just means we have a MAX_NUM_OF_SPACES constant at the top of the confluence runner module, but defer if there's something more elegant.

also this may be a bigger question of how we do this for all of these. if so, probably better for another PR..but also not necessarily the most urgent